htdocs/projet/element.php

```diff
@@ -620,7 +620,7 @@
 
 		$elementarray = $object->get_element_list($key, $tablename, $datefieldname, $dates, $datee, !empty($project_field) ? $project_field : 'fk_projet');
 
-		if (count($elementarray) > 0 && is_array($elementarray))
+		if (is_array($elementarray) && count($elementarray) > 0)
```

Comment:
Hi @frederic34 

"! empty" is better of "count"

http://webjawns.com/2014/04/php-benchmark-count-vs-empty/

Reply:
and if $elementarray is not an array?

Reply:
@frederic34 
you can check if is an array : 

`if (is_array($elementarray) && ! empty($elementarray))`